backend/compact-connect/lambdas/python/compact-configuration/tests/function/test_compact_configuration_api.py

```diff
@@ -0,0 +1,89 @@
+import json
+from decimal import Decimal
+
+from moto import mock_aws
+
+from . import TstFunction
+
+
+def generate_test_event(method: str, resource: str) -> dict:
+    with open('../common/tests/resources/api-event.json') as f:
+        event = json.load(f)
+        event['httpMethod'] = method
+        event['resource'] = resource
+        event['pathParameters'] = {
+            'compact': 'aslp',
+        }
+
+    return event
+
+
+def load_test_jurisdiction(compact_configuration_table, jurisdiction_overrides: dict):
+    with open('../common/tests/resources/dynamo/jurisdiction.json') as f:
+        record = json.load(f, parse_float=Decimal)
+
+    record.update(jurisdiction_overrides)
+    compact_configuration_table.put_item(Item=record)
+
+    # return record for optional usage in tests
+    return record
+
+
+@mock_aws
+class TestGetCompactJurisdictions(TstFunction):
+    """Test suite for get compact jurisdiction endpoints."""
+
+    def test_get_compact_jurisdictions_returns_empty_list_if_no_active_jurisdictions(self):
```

Comment:
Since you're handling HTTP method routing in the entrypoint, you should probably throw in a test for a POST or something to verify the invalid request case, right?